frontend/integration-tests/tests/crd-extensions.scenario.ts

```diff
@@ -0,0 +1,202 @@
+import { $, browser, by, element, ExpectedConditions as until } from 'protractor';
+import * as _ from 'lodash';
+import { safeDump, safeLoad } from 'js-yaml';
+
+import { appHost, testName, checkLogs, checkErrors } from '../protractor.conf';
+import * as crudView from '../views/crud.view';
+import * as yamlView from '../views/yaml.view';
+
+describe('CRD extensions', () => {
+  afterEach(() => {
+    checkLogs();
+    checkErrors();
+  });
+
+  describe('ConsoleClIDownload CRD', () => {
+    const crd = 'ConsoleCLIDownload';
+    const name = `${testName}-ccd`;
+    // cannot use default YAML template since it contains new lines
+    // in the description and that breaks with safeLoad
```

Comment:
We should follow up o this. I'm worried there is a bug here somewhere. It seems to work correctly in my testing, though.

Reply:
I'm guessing it's a bug in js-yaml.  The issue I was seeing was `safeLoad` was tripping over [the new line in the sample template](https://github.com/openshift/console/blob/master/frontend/public/models/yaml-templates.ts#L829) since safeLoad reorders the YAML.